### ISSUE TITLE
No save Labels on trelloApi.updateCard(itemCard);

### ISSUE THREAD

**Issue by omzevr** [taskadapter/trello-java-wrapper#20]
Hi,

When I tried to do the follow statement:

trelloApi.updateCard(itemCard);

The card lost theirs Labels.

In my case, the object "itemCard" is coming from trelloApi.getBoardCards(idBoard) statement and all data is correct until I do the update.

Regards

Omar

**Comment by vitaliypo** [comment]
Indeed, any ```card.update()``` call erases Label from card on Trello board. The only workaround is to save Labels before update() and use them to send correct request by youself: https://developers.trello.com/reference/#cardsidlabels
Tip: this api won't create label duplicate if you use existing parameters (name, color), so you can use it both for creating new labels and assign already existing ones to the card.

**Comment by bryanchug** [comment]
Adding `idLabels` to `Card` should fix the issue.
